### ISSUE TITLE
BYO Moodle release Documentation for Templates

### ISSUE THREAD

**Issue by nimojg-microsoft** [Azure/Moodle#122]
In several conversations customers have asked for clarification on how to use the ARM template with their _own_ Moodle releases. Can we provide documentation on this? (If we don't already)

**Comment by rgardler** [comment]
I don't think this is supported at this time but it would be relatively easy to implement. It's not simply a documentation effort however. We'd be happy to help anyone facing this issue with guidance on delivering a PR, but it's not likely something we will prioritize as our goal is to only use upstream Moodle at this time.
